src/features/hover.zig

```diff
@@ -122,7 +129,11 @@ pub fn hoverSymbol(
             try writer.print("[{s}]({s}#L{d})", .{ ref.str, ref.handle.uri, line });
         }
     } else {
-        try writer.print("{s} ({s})", .{ def_str, resolved_type_str });
+        if (is_fn) {
+            try writer.print("{s}", .{def_str});
+        } else {
+            try writer.print("{s} ({s})", .{ def_str, resolved_type_str });
```

Comment:
I know this is unrelated to your fix but while were modifying hover. I think there should be a line break between the definition string and resolved type just like in the markdown version.

Reply:
[diff](https://github.com/zigtools/zls/compare/a77cac1ba3ba8cd66d471285f835c8432a33ec05..5d84a617b9bd714735bc45f68098ef210ae58edb)